repository/src/main/java/org/pentaho/platform/repository/DatabaseHelper.java

```diff
@@ -29,12 +29,12 @@
 import org.pentaho.di.core.encryption.Encr;
 import org.pentaho.platform.api.repository2.unified.data.node.DataNode;
 import org.pentaho.platform.api.repository2.unified.data.node.DataProperty;
+import org.pentaho.platform.util.StringUtil;
 
 import java.io.Serializable;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 import java.util.Map.Entry;
-import java.util.Set;
+import java.util.stream.Collectors;
```

Comment:
Please validate and remove unsed imports

Reply:
Imports optimized